app/controllers/response_controller.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Why do you need this?

Reply:
That was added automatically by rubocop extension. Removing it.